src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -211,42 +211,25 @@ class CreatePipelineRun extends React.Component {
     // Send API request to create PipelineRun
     const pipelineRef = this.getPipelineInfo(PIPELINE_REF);
     const namespace = this.getPipelineInfo(NAMESPACE);
-    const payload = {
-      metadata: {
-        name: `${pipelineRef}-run-${Date.now()}`
-      },
-      spec: {
-        pipelineRef: {
-          name: pipelineRef
-        },
-        resources: Object.keys(this.state.resources).map(name => ({
-          name,
-          resourceRef: { name: this.state.resources[name] }
-        })),
-        params: Object.keys(this.state.params).map(name => ({
-          name,
-          value: this.state.params[name]
-        })),
-        serviceAccount: this.state.serviceAccount,
-        timeout: this.state.timeout
-      }
-    };
-    // TODO: waiting for backend API endpoint to create arbitrary PipelineRun
-    const promise = createPipelineRun(payload, namespace);
-    promise
-      .then(headers => {
-        const url = headers.get('Content-Location');
-        const pipelineRunName = url.substring(url.lastIndexOf('/') + 1);
-        const finalURL = `/namespaces/${namespace}/pipelines/${pipelineRef}/runs/${pipelineRunName}`;
+    const { params, resources, serviceAccount, timeout } = this.state;
+    createPipelineRunAtProxy({
+      namespace,
+      pipelineRef,
```

Comment:
this should be `pipelineName`, is there a test we could add/update to catch this type of mismatch?

Reply:
Sorry, I forgot to update that file. I also updated the `CreatePipelineRun` test to catch this error.